app/models/supported_ssg.rb

```diff
@@ -15,25 +15,34 @@ def ref_id
     ".content_benchmark_#{OS_NAME}-#{os_major_version}"
   end
 
+  def comparable_version
```

Comment:
If you're introducing this helper method, we probably want to use it across the repo where `Gem::Version˙ is being already used.

Reply:
This is mostly to use a shortened version for mapping and sorting.

Reply:
Yeah, if you do a fulltext search, it's used on multiple places

Reply:
Do you have a recommendation where to move it?

Reply:
I was aiming towards `ApplicationRecord` but now I see that the `SupportedSsg` is a `Struct` :confused: having a separate module that we include in both sounds like an overkill. So I guess it can stay as it is ... sorry.